src/transformation/visitors/return.ts

```diff
@@ -1,12 +1,51 @@
 import * as ts from "typescript";
 import * as lua from "../../LuaAST";
-import { FunctionVisitor } from "../context";
+import { FunctionVisitor, TransformationContext } from "../context";
 import { isInTupleReturnFunction, isTupleReturnCall } from "../utils/annotations";
 import { validateAssignment } from "../utils/assignment-validation";
 import { createUnpackCall, wrapInTable } from "../utils/lua-ast";
 import { ScopeType, walkScopesUp } from "../utils/scope";
 import { isArrayType } from "../utils/typescript";
 
+function transformExpressionToTupleExpressions(
```

Comment:
`TupleExpressions` doesn't make much sense for me. Maybe `transformExpressionsInReturn`, and let it handle https://github.com/hazzard993/TypescriptToLua/blob/update-typescript/src/transformation/visitors/return.ts#L71-L75 too, so that both usages would be simplified to `lua.createReturnStatement(transformExpressionsInReturn(...))` and `const results = transformExpressionsInReturn(...)`?

```ts
if (!isInTupleReturnFunction(context, node)) {
	return [context.transformExpression(statement.expression)];
}

let results: lua.Expression[];
// ...
```